.pre-commit-config.yaml

```diff
@@ -59,20 +59,20 @@ repos:
 
   # Clang-format the C++ part of the code base automatically
   - repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: v14.0.6
+    rev: v16.0.0
     hooks:
       - id: clang-format
         types_or: [c++, c, cuda]
 
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.259
+    rev: v0.0.260
     hooks:
       - id: ruff
         args: ["--fix"]
 
   # Run code formatting with Black
   - repo: https://github.com/psf/black
-    rev: 23.1.0 # Keep in sync with blacken-docs
+    rev: 23.3.0 # Keep in sync with blacken-docs
```

Comment:
Make sure to respect the comment here ;o) (you are missing an update to the `blacken-docs` hook)

Reply:
Good catch, I totally missed that this was autoupdated :see_no_evil: